clientd3d/d3drender.h

```diff
@@ -178,10 +178,18 @@ bool ShouldRenderInCurrentPass(bool transparent_pass, bool isTransparent);
 float FovHorizontal(long width);
 float FovVertical(long height);
 
-bool isFogEnabled();
+int getD3dRenderThreshold();
+
 // Returns the max shading range (FINENESS-shade_amount) to FINENESS
 long getShadeAmount();
+
+bool IsManagedTexturesEnabled();

```

Comment:
Make capitalization consistent between IsManagedTexturesEnabled and isFogEnabled.